torch/testing/_internal/common_utils.py

```diff
@@ -115,10 +115,6 @@ def is_arch(arch_list):
         if gfx_arch in arch_list:
             return True
     return False
```

Comment:
@pragupta We should track the upstreaming of this patch in one of our stories. cc @iupaikov-amd